app/javascript/pages/mygfw/component.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import MyGFWLogin from 'components/mygfw-login';
+import Loader from 'components/ui/loader';
+import Button from 'components/ui/button';
+import ShareModal from 'components/modals/share';
+import ProfileModal from 'components/modals/profile';
+
+import AreasProvider from 'providers/areas-provider';
+import CountryDataProvider from 'providers/country-data-provider';
+
+import SaveAOIModal from 'components/modals/save-aoi';
+import UserProfile from './components/user-profile';
+import AreasTable from './components/areas-table';
+import NoAreasImage from './assets/no-areas@2x.png';
```

Comment:
check the assets, this one looks out of place here

Reply:
Yes, I need fausto to come back, as the export from invision is unusable.

Reply:
As that is a long way off I am going to leave this for now.